meta_review/handler.py

```diff
@@ -0,0 +1,499 @@
+from dateutil import parser
+import json
+import logging
+import os
+
+from django.utils import timezone
+
+from meta_review.dump_to_db import dump_all
+from meta_review.load_from_db import load_all
+from meta_review.models import Participant
+
+
+def parse_time(time):
+    """
+    Parse string to datetime.
+
+    :param time: a string represents time, e.g. 2018-05-09T11:19:26Z
+    :return: an offset-aware datetime object
+    """
+    if time is None:
+        return None
+
+    return parser.parse(time)
+
+
+class MetaReviewHandler:
+    """
+    Handle meta-review system.
+
+    This is the class responsible for scraping provided information (reviews,
+    reactions), processing them and dumping into Django database.
+    """
+
+    # coefficients of the scoring formula
+    C1, C2 = 0.05, 0.2
```

Comment:
better names?

Reply:
I'm pretty bad at naming. What about `BONUS_GIVE_POS` and `BONUS_GIVE_NEG`?